posts/inside-rust/2020-08-12-changes-to-x-py-defaults.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Changes to x.py defaults"
+author: Joshua Nelson
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Recently, the defaults for [`x.py`], the tool used to [bootstrap] the Rust compiler from source, changed. If you regularly contribute to Rust, this might affect your workflow.
+
+## What changes were made?
+
+- The default stage is now dependent on the subcommand:
+  + `dist`: stage 2
+  + `build`: stage 1
+  + `test`: stage 1
+  + `doc`: stage 0
+
+- stage1 `rustc` artifacts are no longer built by `x.py build --stage 1`. To get the old behavior back, use `x.py build --stage 1 src/rustc`. The new behavior for `build --stage 1` builds everything except `rustc`, which includes the standard library, `rustdoc`, and various other tools (if the tools are enabled).
+
+- `debuginfo` now defaults to `1` when `debug = true`. Previously, the default was 2.
+
+## Why were the changes made?
+
+Previously, `x.py build` would build `rustc` three separate times:
+
+1. `build/stage0-rustc`
+2. `build/stage1-rustc`
+3. `build/stage2-rustc`
+
+Normally, contributors only want to build the compiler once, which lets them test their changes quickly. After this change, that's now the case:
```

Comment:
Maybe it would be clearer as "that's now the default"?

Reply:
```suggestion
Normally, contributors only want to build the compiler once, which lets them test their changes quickly. After this change, that's now the default:
```